coverage/.resultset.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
you need to add coverage to the `.gitignore` and remove the git cache

http://www.codeblocq.com/2016/01/Untrack-files-already-added-to-git-repository-based-on-gitignore/